src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -17,13 +22,31 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public DeleteCommand parse(String args) throws ParseException {
+        System.out.println(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_PHONE, PREFIX_INDEX);
+
+        if ((!arePrefixesPresent(argMultimap, PREFIX_PHONE) && !arePrefixesPresent(argMultimap, PREFIX_INDEX))
```

Comment:
why not use anyMatch in arePrefixesPresent and use varargs instead of the &&

Reply:
actually why not move both checks in to the try? so:

if arePrefixesPresent(argMultimap, PREFIX_PHONE)
 |_> handling
else if arePrefixesPresent(argMultimap, PREFIX_INDEX)
|_> handling
else
|_> throw ParseException